### ISSUE TITLE
NTR: Catecholamines low in blood or urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8877]
New term request Catecholamines low in blood or urine (segal_220201182341):   UMLS: 0860772

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000482
- label: Decreased circulating catecholamine concentration
- parents: [Abnormality of circulating catecholamine level (HP:0012099)](https://hpo.jax.org/app/browse/term/HP:0012099)
- pmid: 
- def: Concentration of a catecholamine in the blood circulation below the lower limit of normal.
- comment: The group of the catecholamines includes epinephrine (adrenaline), norepinephrine (noradrenaline), and dopamine.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8877
- synonyms:
